src/Bookings.js

```diff
@@ -1,9 +1,16 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import Search from "./Search.js";
 import SearchResults from "./SearchResults.js";
-import FakeBookings from "./data/fakeBookings.json";
+// import FakeBookings from "./data/fakeBookings.json";
```

Comment:
I'd recommend removing commented out code, as it can get confusing! It'll still be there in your git history, if you ever need to look at it again!